splash/browser_tab.py

```diff
@@ -620,13 +620,6 @@ def wait_for_resume(self, js_source, callback, errback, timeout):
         one argument. The first argument will be an object with `resume()`
         and `error()` methods. The code _must_ call one of these functions
         before the timeout or else it will be canceled.
-
-        Note: this cleans up the JavaScript global variable that it creates,
-        but QT seems to notice when a JS GV is deleted and it destroys the
-        underlying C++ object. Therefore, we can only delete the JS GV _after_
-        the user's code has called us back. This should change in QT5, since
-        it will then be possible to specify a different object ownership
-        policy when calling addToJavaScriptWindowObject().
```

Comment:
@mehaase - do you recall details about this issue? I can't reproduce it in qt5.5.1, but I wonder if that's because the issue is fixed or because examples are not complex enough to trigger the bug. Did Splash testing suite segfault, or did you have other segfaulting examples back then?